modules/sdk-core/src/bitgo/lightning/iLightning.ts

```diff
@@ -149,6 +155,18 @@ export const GetInvoicesResponse = t.array(InvoiceInfo);
 // eslint-disable-next-line no-redeclare
 export type GetInvoicesResponse = t.TypeOf<typeof GetInvoicesResponse>;
 
+export const LnurlPayResponse = t.strict({
+  tag: t.literal('payRequest'),
+  callback: t.string,
+  maxSendable: t.number,
+  minSendable: t.number,
+  metadata: t.string,
+  domain: t.union([t.string, t.undefined]),
```

Comment:
I don't think `domain` is one of the elements on the first response? Just going off of what's described here https://github.com/fiatjaf/lnurl-rfc/blob/luds/06.md#pay-to-static-qrnfclink. You should just parse it from the URL.

Reply:
lnurl spec 4. specifies that we should "display" max can send, min can send, metadata, and domain. I thought it would be reasonable to include them in the decode part

Reply:
OK, I just misinterpreted how the type was being used. That's fine then. Why would it be undefined though? We should always have a domain because we'll always have a url.